setup.py

```diff
@@ -15,15 +15,16 @@
     history = history_file.read().decode('utf8')
 
 requirements = [
-    "marshmallow>=3.10.0",
+    "marshmallow>=3.10.0,<4.0",
```

Comment:
Since we're about to release a major version, it would be nice to support marshmallow 4.

Do you think you can investigate whether we can support both 3 and 4? If it is too difficult, we may drop MA3.

Not a blocker at all, we can do that in a later PR.